src/dfm/projects/production/model_training/scripts/yamls/continue-mistral-7b.yaml

```diff
@@ -0,0 +1,149 @@
+# Continue-pretrain a Mistral-7b model
+# This is NOT a finetuning run
+max_seq_len: 2048
+global_seed: 17
+data_local: /scratch/project_465000670/da-gigaword-mds
+data_remote: # If blank, files must be present in data_local
+pretrained_model_name: mistralai/Mistral-7B-v0.1 #meta-llama/Llama-2-7b-hf
+tokenizer_name: mistralai/Mistral-7B-v0.1 #mistralai/Mistral-7B-v0.1
+
+# Run Name
+run_name: continue-mistral # If left blank, will be read from env var $RUN_NAME
+
+# HF
+model:
+ name: hf_causal_lm
+ pretrained: true
+ pretrained_model_name_or_path: ${pretrained_model_name}
+ token: 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
+ use_flash_attention_2: true
+ config_overrides:
+   #sliding_window: ${max_seq_len}
```

Comment:
?

Reply:
I'm actually unsure if the sliding window size should be changed like this.

Reply:
Any reason we would overwrite it?

Reply:
Sure def. agree